runelite-client/src/main/java/net/runelite/client/ui/overlay/components/InfoBoxComponent.java

```diff
@@ -34,12 +35,19 @@
 import java.util.Objects;
 import javax.annotation.Nullable;
 import lombok.Setter;
+import net.runelite.client.ui.FontManager;
 import net.runelite.client.ui.overlay.RenderableEntity;
 
 public class InfoBoxComponent implements RenderableEntity
 {
-	private static final int BOX_SIZE = 35;
-	private static final int SEPARATOR = 2;
+	private final int boxSize;
```

Comment:
This should be setter instead of being in constructor with default values set to 35 and 2 respectively